src/comm/LinkManager.cc

```diff
@@ -673,8 +674,19 @@ void LinkManager::_updateAutoConnectLinks(void)
         _autoConnectRTKPort.clear();
     }
 #endif
+#else
```

Comment:
@DonLakeFlyer The "Check for RTK GPS connection gone" on line 669 results in the RTK GPS stopping immediately after configuration.  Is there a good way to handle the disconnection that you guys can recommend?